edweiss-app/components/memento/CardListDisplayComponent.tsx

```diff
@@ -85,9 +87,12 @@ export const CardListDisplay: React.FC<{
                     {/*<TText bold color='text' ellipsizeMode='tail' numberOfLines={1}>
                         {card.question}
                     </TText>*/}
-                    <RichText px={'sm'} color='text'>
+                    <RichText px={'sm'} color={isSelected ? 'crust' : 'text'}>
                         {card.question}
                     </RichText>
+                    <TText mt={'sm'} mb='md' color='subtext0' size={'sm'}>
+                        Created by: {userIdToName(card.ownerID) ?? 'who the heck is this?'}
```

Comment:
I never seen the `??`, what does it do. Just a curiosity question.

Reply:
The ?? operator in TypeScript is the nullish coalescing operator. It is used to provide a default value when the value on the left-hand side is null or undefined. This operator is particularly useful for handling optional values or initializing variables with a fallback.